test/testBearingRange2D.jl

```diff
@@ -54,13 +54,7 @@ xi = getPointIdentity(Pose2)
 li = zeros(2); li[1] = 20.0;
 zi = (zeros(2),); zi[1][2] = 20.0
 
-
-res = testFactorResidualBinary( p2br, 
-                                Pose2, 
-                                Point2, 
-                                xi, 
-                                li, 
-                                zi )
+res = calcFactorResidualTemporary( p2br, zi, (Pose2, Point2), (xi, li)) 
```

Comment:
Trying to add suggested changes
```suggestion
res = calcFactorResidualTemporary( p2br, (Pose2, Point2), zi, (xi, li)) 
```